src/core/Form/Select/SingleSelect/SingleSelect.tsx

```diff
@@ -211,6 +211,9 @@ class BaseSingleSelect<T> extends Component<
 
   private focusToInputAndSelectText = () => {
     if (!!this.filterInputRef && this.filterInputRef.current) {
+      this.setState({
+        showPopover: true,
+      });
```

Comment:
Could there be a cleaner way to do this? With only this in place, for example in the next method `focusToInputAndCloseMenu` this is now called before the menu is closed again that method. Seems like opening the menu does not suit all the situations where this method is called even though it does solve the problem in question.

Reply:
Fixed.